bitnami/node-exporter/templates/psp-clusterrole.yaml

```diff
@@ -3,7 +3,7 @@
 kind: ClusterRole
```

Comment:
You also need to update the PSP reference in line 20 to use the `namespace` extension

Reply:
Great catch! Changed.